### DISCUSSION TITLE
GoPro MAX Delayed 360 Photo

### DISCUSSION THREAD

**Discussion by Gaberham007** [gopro/labs#730] [Q&A]
Cheers folks,

TL;DR: Looking to take a delayed 360 photo (Flash QR, photo gets snapped after 15sec).

I've recently come across the GoPro Labs QR code functionality, and was delighted by the applications! I'm an avid scuba diver, and I aim to produce some underwater VR content with my MAX and 360 Bubble Housing. The QR codes have been great implementing pre-dive settings (for battery consumption), ideal underwater capture settings, start/end recording, etc. since once the GoPro is in the housing (and underwater) button manipulation is impossible. Previously it was start recording, put in the housing, start the dive, a bunch of awkward footage, and hope the battery lasts for the duration.

What I would like to accomplish is to flash a QR code that prompts a 360 photo after a ~10-15sec delay. The idea is to flash the QR code, then get into a super cool position that will let people know I ~am~ cool, then a normal 360 photo is captured.

I've tried a few manipulations of "!15NmP!S", but the issue is that the GoPro gets stuck on "Processing" until I do a hard reset.

I have an adequate SD card. When the capture is taken, the GoPro starts whatever delay was set in the preferences; i.e. after 15sec, it starts a 10sec countdown then captures. I've also tried with an !E to end the capture, but the same issue arises of the device getting stuck in "Processing".

Anyways, any suggestions on a QR code that prompts the device to take a 360 photo? Thanks in advance!

Also, apologizes if this is very elementary and I'm missing some basic functions. Reading through previous discussions, I found they were all focus on video settings.

**Comment by dnewman-gpsw** [comment]
try mP!9B!3B!1B!1B!S

Any pause greater than 9 seconds will shutdown the camera.  !B <- is a blink. So this script will pause blinks that get more rapid as the countdown closes.
![image](https://github.com/gopro/labs/assets/10856084/416fa793-78af-4ba9-b549-52a8dc14b08a)

**Reply by Gaberham007** [reply]
Thanks for the feedback! The script did as you described; however, it seems that I may be dealing with a different issue. Even when attempting a manual capture, the GoPro gets locked up on the "Processing" screen. I think it may be an issue with the GoPro MAX and the Labs Firmware, as it didn't occur prior to installing the firmware. I intend to rest to factory firmware and see if the same issue persists - unless there are any other thoughts.

Thanks again!

**Reply by dnewman-gpsw** [reply]
Not Labs firmware, as my Labs MAX is working fine, but a factory reset is a good idea when things are running weird.

**Reply by Gaberham007** [reply]
I've taken these steps: !Reset to clear all QR code metadata, removed Labs firmware and put on original Max firmware, formatted the SD card, and reinstalled Labs Firmware

All without success. My camera still stalls at "processing" when attempting to take a 360 photo.

So it seems to be an issue outside of Labs. I'll reach out to customer support and see what they recommend. Open to any suggestions though. Thank you!

**Reply by Gaberham007** [reply]
I stand corrected. I just did a factory reset as well, and now the camera is taking 360 photos. Maybe there were some incompatible settings selected?

However, the QR code you previously provided instantaneously take a 360 photo. No delay/timer involved. Still open to suggestions. Thank you!
